modin/pandas/test/test_general.py

```diff
@@ -801,3 +801,22 @@ def test_empty_dataframe():
 def test_empty_series():
     s = pd.Series([])
     pd.to_numeric(s)
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+def test_to_timedelta(data):
```

Comment:
Could you add another test case where we pass:

- a list-like of ints, like `[1]`
- a list-like of invalid strings, like `['a']`. This should raise `ValueError`.
- a valid scalar, like `1`
- an invalid scalar, like '"a"'

We can do something like `@pytest.mark.parametrize('arg', [[1, 2], ["a"], 1, "a"], ids=["list_of_ints", "list_of_invalid_strings", "scalar", "invalid_scalar"])`